dist/OutSystemsUI.js

```diff
@@ -19829,10 +19829,10 @@ var Providers;
                     }
                     _setIsDisabled(isDisabled) {
                         if (isDisabled) {
-                            OSFramework.OSUI.Helper.Dom.Disable(this._rangeSliderProviderElem);
+                            this.provider.disable();
```

Comment:
Suggestion: can we do a inline if like isDisabled ? this.provider.disable() : this.provider.enable()?

Reply:
Fixed!